Developers/Writing_Bears.rst

```diff
@@ -150,6 +150,42 @@ This will show the user a bunch of information related to the bear like:
 - A description of what the bear does - The sections which uses it - The
 settings it uses (optional and required)
 
+.. note::
+
+    The bears are not yet installed. We still have to specify
+    the bear directory using ``-d`` or ``--bear-dirs`` flag.
+
+
+Installing the New coala-bears Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To install new coala-bear locally make sure :doc:`Bear Installation Tool
+(cib) <./Bear_Installation_Tool>` is installed in our environment.
```

Comment:
this tool is experimental, you dont need it to install the bears, you can just run pip3 install coala-bears